scripts/update_avalanchego_version.sh

```diff
@@ -4,56 +4,48 @@
 
 set -euo pipefail
 
-# Check if a version argument is provided
```

Comment:
This seems to be failing when I call it with the current commit of AvalancheGo locally (and using `nix develop`), but it fails with:

```
./scripts/update_avalanchego_version.sh 8551e1d96221
Ensuring AvalancheGo version to 8551e1d96221 in go.mod
Ensuring AvalancheGo version 8551e1d96221 in /Users/aaronbuchwald/go/src/github.com/ava-labs/hypersdk/go.mod
curl: option : blank argument where content is expected
curl: try 'curl --help' or 'curl --manual' for more information
```

Am I calling it incorrectly? Maybe we can add `# Usage ...` at the top of the file for future reference (not necessary for this PR)